WordPress/Classes/Services/NotificationSyncMediator.swift

```diff
@@ -257,15 +282,19 @@ class NotificationSyncMediator {
     /// Invalidates the local cache for the notification with the specified ID.
     ///
     func invalidateCacheForNotification(with noteID: String) {
+        invalidateCacheForNotifications(with: [noteID])
+    }
+
+    /// Invalidates the local cache for all the notifications with specified ID's in the array.
+    ///
+    func invalidateCacheForNotifications(with noteIDs: [String]) {
```

Comment:
Nitpick but I'd probably switch the `with` for a `_` here, as I think it reads better when calling the method.